.travis.yml

```diff
@@ -19,6 +21,7 @@ install:
   - sudo iptables -t nat -L >/dev/null
   - sudo iptables -t filter -P FORWARD ACCEPT
   - sudo /usr/bin/python3 ./tests/tunneldigger.py --setup
+  - ( cd broker; docker build --tag wlanslovenija/tunneldigger-broker . )
```

Comment:
So this just checks if the docker image builds, right, but does not run it?

If yes, no need to do that in *each* Travis job.